llm_api/openai_assistant.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+from openai.types.chat import ChatCompletion, ChatCompletionMessageParam
+
+from .iassistant import IAssistant
+from openai import OpenAI
+import tiktoken
+
+
+MODEL_NAME = "gpt-3.5-turbo"
+enc = tiktoken.encoding_for_model(MODEL_NAME)
+
+
+def _get_response(response: ChatCompletion) -> str:
```

Comment:
Should it not be an  OpenAIAssistant method (since it is not used in any other place, by design)?

Reply:
IMO no because this method does not depend on the internal variables in the class so the method may be moved to the outer scope